BaSP-a2022/ayom-a-trackgenix-app
# BaSP-TG-146: Create Footer Component

- Create footer component
- Add footer to routes index
- Add social media buttons/links
- Add contact data
- Add contact data

QA:

To test this PR you should see the footer always in the application, when click social media buttons must redirect to respective page.

Note: When click on trackgenix logo should redirect to home/landing page but it doesn't, this will be fixed in the Landing Page PR!
